### PR TITLE
upgrade openpyxl

### PR DESCRIPTION
**Pull request by nickpell**
https://openpyxl.readthedocs.io/en/stable/changes.html

### CODE REVIEW

**Review by emord — APPROVED** [review]

**Comment by nickpell** [comment]
@emord @orangejenny Could I get a second review on this PR?  I had to make some changes to handle incompatibilities with the updated API

File: corehq/ex-submodules/couchexport/writers.py
Comment (emord) [inline corehq/ex-submodules/couchexport/writers.py]: I'm not sure what "default sheet" from the commit means. Maybe this could use a comment to say why it's necessary?

File: corehq/util/workbook_json/excel.py
Comment (emord) [inline corehq/util/workbook_json/excel.py]: Does this "just work" now? is there a test for whatever `force=True` was meant to fix?

File: corehq/ex-submodules/couchexport/writers.py
Comment (nickpell) [reply corehq/ex-submodules/couchexport/writers.py]: Updated - when I passed the ```write_only``` kwarg a default sheet isn't created anymore

File: corehq/util/workbook_json/excel.py
Comment (nickpell) [reply corehq/util/workbook_json/excel.py]: Added back ```read_only``` in place of ```use_iterators```, which then means that ```calculate_dimension``` expects ```force=True```.  (has test coverage)

**Review by emord — APPROVED** [review]

**Pull request merged**
